tests/attr/neuron/test_neuron_conductance.py

```diff
@@ -43,6 +43,13 @@ def test_simple_conductance_input_linear1(self) -> None:
         inp = torch.tensor([[0.0, 100.0, 0.0]])
         self._conductance_input_test_assert(net, net.linear1, inp, 0, [0.0, 90.0, 0.0])
 
+    def test_simple_conductance_input_linear1_selector_fn(self) -> None:
+        net = BasicModel_MultiLayer()
+        inp = torch.tensor([[0.0, 100.0, 0.0]])
+        self._conductance_input_test_assert(
+            net, net.linear1, inp, lambda x: x[:, 0], [0.0, 90.0, 0.0]
+        )
+
```

Comment:
If there is a test model that has a custom layer that returns a tuple, could we, please, add a test for that too.

Reply:
Yes, I think both the other 2 added tests cover this case. Passing multi_input_module=True uses a custom layer MultiRelu, which returns a tuple.